src/state/learnocaml_data.ml

```diff
@@ -444,6 +444,52 @@ module Exercise = struct
       in
       by_status_explicit base a |> STM.bindings
 
+    let three_way_merge ~ancestor ~theirs ~ours =
```

Comment:
I am not sure to understand this function: can the merge encounter conflicts? Can it fail?